ui/component/userFirstChannel/view.jsx

```diff
@@ -63,6 +143,40 @@ function UserFirstChannel(props: Props) {
         }
         actions={
           <Form onSubmit={handleCreateChannel}>
```

Comment:
I don't think we want to prevent it. The idea was to encourage new users to pick one to get less astronauts overall. Making an unnecessary field mandatory might lead to even more "issues"...